TROUBLESHOOT.md

```
@@ -0,0 +1,478 @@
+npx prisma migrate dev
+```
+
+If migration history becomes inconsistent:
+
+```bash
+npx prisma migrate resolve --applied <migration_name>
+```
+
+Replace:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)